packages/ts/generator-typescript-cli/test/GeneratorIO.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { expect } from 'chai';
+
+// TODO: This is a stub. We need to add real tests
```

Comment:
Do we expect to add these missing tests in the scope of this PR or later? If later, there should be another issue about these.

Reply:
They should go later. The stub is added only to satisfy the test runner (otherwise, it fails).